### PR TITLE
pcds-5.9.1

### PR DESCRIPTION
**Pull request by ZLLentz**
Small release, less ambitious than #345 
I grabbed the incr env (py39) from GHA after it passed, then made that build the current-env.
See https://jira.slac.stanford.edu/browse/ECS-6281

I think the degraded torch versions from from xopt's dependencies

Added the Following Packages
----------------------------

- xopt

PCDS Package Updates
--------------------

|   Package   |  Old   |  New   |                       Release Notes                        |
| :---------: | :----: | :----: | :--------------------------------------------------------: |
|     atef    | 1.5.0  | 1.5.1  |    https://github.com/pcdshub/atef/releases/tag/v1.5.1     |
|  lightpath  | 1.0.4  | 1.0.5  |  https://github.com/pcdshub/lightpath/releases/tag/v1.0.5  |
| pcdsdevices | 8.5.0  | 8.6.0  | https://github.com/pcdshub/pcdsdevices/releases/tag/v8.6.0 |
|    pytmc    | 2.16.0 | 2.17.0 |   https://github.com/pcdshub/pytmc/releases/tag/v2.17.0    |

Packages With Degraded Versions
-------------------------------

|     Package     |  Old   |  New   |
| :-------------: | :----: | :----: |
|     botorch     | 0.11.3 | 0.10.0 |
|     gpytorch    |  1.12  |  1.11  |
| linear_operator | 0.5.2  | 0.5.1  |

Added the Following Dependencies
--------------------------------

- deap (required by xopt)
- libgcc (required by cryptography, deap, libgcc-ng, libstdcxx, openssl, which are used in atef, bluesky, bluesky-live, databroker, happi, hklpy, ipython, jupyter, ophyd, pandas, pmpsui, scikit-image, suitcase-tiff, xarray)
- libstdcxx (required by deap, libstdcxx-ng)
- lightpath-base (required by lightpath, pcdsdevices)
- orjson (required by hklpy, xopt)

### CODE REVIEW

**Comment by ZLLentz** [comment]
Confirmed that the annoying warning is gone when running locally
If/after the build passes I'll paste in the release notes and ask for a review

**Comment by ZLLentz** [comment]
The various failures aside from py312 are for a security CVE that doesn't apply to the as-built env.
py312 is still stuck on p4p

**Review by tangkong — APPROVED** [review]
Seems un-objectionable to me 👍

File: envs/pcds/env.yaml [hunk 1]
```diff
@@ -878,4 +884,4 @@ dependencies:
       - python-vxi11==0.9
       - slac-alarm-manager==1.3.0
       - tc-release==0.2.6
-prefix: /cds/home/z/zlentz/miniconda3/envs/pcds-5.9.0
+prefix: /home/runner/miniconda/envs/pcds-5.9.1
```
[the remark this answers is not in the document]
Comment (ZLLentz) [reply envs/pcds/env.yaml:887]: haha, yeah it depends on whether I build the env myself or let the runner do it
It doesn't actually affect the installation of the env from this file

File: envs/pcds/conda-packages.txt [hunk 2]
```diff
@@ -122,6 +122,8 @@ bluesky-base=1.10.0
 databroker=1.2.5
 # ipython changes make scientists upset and can break experiments
 ipython=8.4.0
+# newer line_profiler requires newer ipython
+line_profiler=4.1.1
```
[the remark this answers is not in the document]
Comment (ZLLentz) [reply envs/pcds/conda-packages.txt:126]: For documentation purposes:

line_profiler `4.1.3` has:
```
constraints :
  - ipython >=8.14.0
```
which I think is what run_constrained looks like when you do `conda search --info`

`4.1.1` has:
```
dependencies:
  - ipython >=0.13
```

`4.1.2` did not make it onto conda

**Pull request merged**